util/dockernode.go

```diff
@@ -39,22 +39,33 @@ func (dn *DockerNode) Start(args []string) error {
 	}
 
 	id := bytes.TrimSpace(out)
+	dn.ID = string(id)
 	idfile := filepath.Join(dn.Dir, "dockerID")
 	err = ioutil.WriteFile(idfile, id, 0664)
+
 	if err != nil {
+    	killContainer(dn.ID)
```

Comment:
nit: this could be git's diff messing up, but I believe `gofmt` may need to be ran to fix indentation issues here and below on line 53

Reply:
Ah, you're right, that one's on me. I'll `go fmt` and fix this.